build.cmd

```diff
@@ -518,8 +518,7 @@ if defined TF_BUILD (
     git fetch --all
 )
 
-set msbuildflags=%_nrswitch% /nologo
-REM set msbuildflags=%_nrswitch% /nologo
+set msbuildflags=%_nrswitch% /nologo /m
```

Comment:
I believe there was a flacky build error with /m flag. I think it was @KevinRansom who removed it.